.devcontainer/docker-compose.override.yaml

```diff
@@ -3,6 +3,9 @@ name: lumigator
 services:
 
   backend:
-    command: ["uv", "run", "uvicorn", "backend.main:app", "--host", "0.0.0.0", "--port", "8000",  "--reload"]
+    build:
```

Comment:
just double-checking, this is for the shipped image, not local dev  right? want to make sure we can still hot-reload the app locally

Reply:
Hey @veekaybee, yep, the override is for local development. I have to reestructure the Dockerfile, and with the new changes there's no need to have it here. Now it's located [here](https://github.com/mozilla-ai/lumigator/blob/f61e79ee1b603527f55d241b293af2866c740fa6/Dockerfile#L21)